docs/guide/export.rst

```diff
@@ -0,0 +1,65 @@
+.. _export:
+
+
+Exporting models
+================
+
+After training an agent, you may want to deploy/use it in an other language
+or framework, like `tensorflowjs <https://github.com/tensorflow/tfjs>`_.
+Stable Baselines3 does not include tools to export models to other frameworks, but
+this document aims to cover parts that are required for exporting along with
+more detailed stories from users of Stable Baselines3.
+
+
+Background
+----------
+
+In Stable Baselines3, the controller is stored inside policies which convert
+observations into actions. Each learning algorithm (e.g. DQN, A2C, SAC) contains
```

Comment:
"Each learning algorithm (e.g. DQN, A2C, SAC) contains one policy". I would replace this with something that clarifies this is the main behaviour policy that decides acts when doing e.g. `model.predict`. E.g:

"Each learning algorithm (e.g. DQN, A2C, SAC) contains a policy which represents the currently learned behaviour, accessible via ``model.policy``. "